.rubocop.yml

```diff
@@ -1,3 +1,10 @@
 # use the shared Yast defaults
 inherit_from:
   /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+Metrics/LineLength:
+  Max: 150
```

Comment:
is it really needed? I hope for new code we can have reasonable long lines. If you need hint how to achieve it just ask.